### ISSUE TITLE
Comments: link to an area should be reformatted to show area's name instead of URL

### ISSUE THREAD

**Issue by dmucli** [theCrag/website#3383]
**What happened?**
When adding a link to an area in a discussion, would be nice if the URL was replaced by the area name. Eg "Bicheno" instead of "https://www.thecrag.com/climbing/australia/bicheno"

Ideally also provide screenshots:
![image](https://user-images.githubusercontent.com/29764214/53839920-d92dcd00-3fec-11e9-8a5b-8967ac417da2.png)

**Comment by lordyavin** [comment]
Are you aware if the markdown notation to set links?

  [link text](link URL)

There is an auto linking function that creates links for route and area names. Put the name of the route or area into single quotation marks. Problem is that this doesn't work everywhere on the page. See #3364

**Comment by dmucli** [comment]
I wasn't aware of that markdown and I guess most users wouldn't either (by the way is there anywhere on the site where this is explained?). 

This 'issue' isn't really urgent or important but would nice to have... as long is it's not too hard to get done.

**Comment by lordyavin** [comment]
https://www.thecrag.com/de/artikel/updatingdescriptions#formatting

**Comment by brendanheywood** [comment]
Yeah the markdown is nerdy and I don't expect most users to know it. We already do some things like this eg swapping out a photo url with the actual photo.

**Comment by georg-d** [comment]
While this is about areas, https://github.com/theCrag/website/issues/2765 is more general.

The opposite approach, so typing the link target's name and let the system create the URL for you, is not easy to handle for non-unique names; see e.g. https://github.com/theCrag/website/issues/3941
